app/controllers/api/v2/registration_controller.rb

```diff
@@ -0,0 +1,64 @@
+module Api
+  module V2
+    class RegistrationController < V2::BaseController
+      include Api::Version2
+      include Foreman::Controller::Registration
+      include Foreman::Controller::Parameters::Host
+
+      rescue_from ActiveRecord::RecordNotFound do |error|
+        logger.info "#{error.message} (#{error.class})"
```

Comment:
Would this  be a suitable use case for `Foreman::Logging.exception` ?

Reply:
yes, and also it should be handled with script as a response (like we do for other errors)

Reply:
It's done same way as it is in https://github.com/theforeman/foreman/blob/324bafdb625b06fe939530ef77534170be855447/app/controllers/api/base_controller.rb#L37:L38
```
    rescue_from ActiveRecord::RecordNotFound do |error|
      logger.info "#{error.message} (#{error.class})"
```

Should I change it also there?

Reply:
ah makes sense, this is kind of expected 404 handling and the traceback is not really useful